SharedRules.DotSettings

```diff
@@ -0,0 +1,1186 @@
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/CustomPatternPlaceholder/=type/@KeyIndexDefined">True</s:Boolean>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/CustomPatternPlaceholder/=type/Properties/=ExactType/@EntryIndexedValue">False</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/CustomPatternPlaceholder/=type/Properties/=Type/@EntryIndexedValue"></s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/CustomPatternPlaceholder/=type/Type/@EntryValue">TypePlaceholder</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/LanguageName/@EntryValue">CSHARP</s:String>
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/MatchCatchClauseWithoutExceptionFilter/@EntryValue">False</s:Boolean>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/SearchPattern/@EntryValue">$type$ $Print$($args$)&#xD;
+{	&#xD;
+	$st$	&#xD;
+}</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/Severity/@EntryValue">SUGGESTION</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=58E9E83BA1F8BF40B58BBDA1E8FDF0BC/SuppressionKey/@EntryValue">DontIncludeNumbers</s:String>
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=5B53D1E26C3D03409F387ABA164C973F/@KeyIndexDefined">True</s:Boolean>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=5B53D1E26C3D03409F387ABA164C973F/Comment/@EntryValue">Post-fix asynchronous methods with Async of TaskAsync (SDCS-1420)</s:String>
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=5B53D1E26C3D03409F387ABA164C973F/CustomPatternPlaceholder/=args/@KeyIndexDefined">True</s:Boolean>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=5B53D1E26C3D03409F387ABA164C973F/CustomPatternPlaceholder/=args/Properties/=Maximal/@EntryIndexedValue">-1</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=5B53D1E26C3D03409F387ABA164C973F/CustomPatternPlaceholder/=args/Properties/=Minimal/@EntryIndexedValue">-1</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=5B53D1E26C3D03409F387ABA164C973F/CustomPatternPlaceholder/=args/Type/@EntryValue">ArgumentPlaceholder</s:String>
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=5B53D1E26C3D03409F387ABA164C973F/CustomPatternPlaceholder/=Book/@KeyIndexDefined">True</s:Boolean>
+	<s:String x:Key="/Default/PatternsAndTemplates/StructuralSearch/Pattern/=5B53D1E26C3D03409F387ABA164C973
```

Comment:
This pattern does not match the related rule https://github.com/Softeq/dotnet-guidelines/blob/master/13_Maintainability.md#avoid-conditions-with-double-negatives-sdcs-1303-2 .

For example this pattern should not be triggered with Any check:
```csharp
var list  = new List<string>();
var isEmpty = !list.Any();
```
As there are no efficient ways to achieve the required behavior, I suggest deleting the pattern.

Reply:
removed